skrub/_reporting/_summarize.py

```diff
@@ -1,14 +1,22 @@
 """Get information and plots for a dataframe, that are used to generate reports."""
 
 from .. import _dataframe as sbd
-from . import _associations, _plotting, _utils
+from . import _associations, _plotting, _sample_table, _utils
 
 _HIGH_CARDINALITY_THRESHOLD = 10
 _SUBSAMPLE_SIZE = 3000
 _N_TOP_ASSOCIATIONS = 20
 
 
-def summarize_dataframe(df, *, order_by=None, with_plots=True, title=None):
+def summarize_dataframe(
+    df,
+    *,
+    order_by=None,
+    with_plots=True,
+    title=None,
+    max_top_slice_size=5,
```

Comment:
Maybe ``top_rows`` and ``bottom_rows`` ? Or ``n_top_rows`` ?
Feels like ``max_top_slice_size`` is kind of long

Reply:
indeed it is long, but there are quite a few "rows" flying around, and we need to distinguish between the maximum number of displayed rows and the actual number so I wanted to be very explicit here. 

note this is a parameter of a private helper, the parameter exposed to the user is just `n_rows`